pygmt/datasets/earth_relief.py

```diff
@@ -10,8 +10,8 @@
 from pygmt.src import grdcut, which
 
 
-@kwargs_to_strings(region="sequence")
-def load_earth_relief(resolution="01d", region=None, registration=None):
+@kwargs_to_strings(convert_bools=False, region="sequence")
+def load_earth_relief(resolution="01d", use_srtm=False, region=None, registration=None):
```

Comment:
If my understanding is right, `use_srtm` with a value of `False` will be removed by `kwargs_to_strings`.

https://github.com/GenericMappingTools/pygmt/blob/6a6171e3efc112f2a8a86088e24caf932149e928/pygmt/helpers/decorators.py#L296-L297

However, if I set `use_srtm` to be `False` and print it within `load_earth_relief`, it is still `False`.